### PR TITLE
Be 27 defaul admin

### PR DESCRIPTION
**Pull request by zafertaskent**
It works now

### CODE REVIEW

File: api/services/passport.js [hunk 1]
```diff
@@ -151,27 +148,39 @@ passport.use(new GoogleStrategy({
...
-    console.log(profile);
-
     process.nextTick(function() {
-
       findByUsername(profile.id, function(err, userdb) {
 
         if (err)
           return done(null, err);
         if (!userdb) {
+	  var role = 1;
```
Comment (adilatilgan) [inline api/services/passport.js]: This code is not formatted, we are using editorconfig. You should find a plugin for your editor and format the code.

File: config/bootstrap.js [hunk 2]
```diff
@@ -14,19 +14,5 @@ module.exports.bootstrap = function(cb) {
...
-    {
-      username: sails.config.admin.username || 'admin',
-      password: sails.config.admin.password || 'admin',
-      role: sails.config.admin.role || 4
-    }
-  ).exec(function createFindCB(error, createdOrFoundRecords) {
-      cb();
-  });
- // cb();
+  cb();
```
Comment (adilatilgan) [inline config/bootstrap.js]: Why did you remove findOrCreate function?

File: config/env/development.js [hunk 3]
```diff
@@ -20,8 +20,8 @@ module.exports = {
   // models: {
   //   connection: 'someMongodbServer'
   // }
-  host: '127.0.0.1',
-  port: 1337,
+  host: '12.33.220',
```
Comment (adilatilgan) [inline config/env/development.js]: What are these they do not look like an IP or port

File: config/bootstrap.js (hunk 2 above)
Comment (zafertaskent) [inline config/bootstrap.js]: We do not need to create an admin account further more. We use emails in config files.

Comment (adilatilgan) [inline config/bootstrap.js]: Oh yeah we talked about that before.
